### DISCUSSION TITLE
vue + vite project Error

### DISCUSSION THREAD

**Discussion by Timson020** [google/model-viewer#4150] [Q&A]
### Description
error msg: 
http://192.168.100.100:9050/webapp/node_modules/.vite/deps/@google_model-viewer.js?v=a018bbe8
network error code: 504

### Version

- model-viewer: v3.0.2

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [x] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Your link doesn't work; can you please explain what the trouble is?
